fractions/logstash/src/main/java/org/wildfly/swarm/logstash/runtime/LogstashCustomizer.java

```diff
@@ -54,11 +57,19 @@ public void customize() {
                     .namedFormatter("logstash")
                     .properties(handlerProps);
 
+            List<String> rootLoggerHandlers = new ArrayList<>();
+
+            RootLogger existingRootLogger = this.logging.subresources().rootLogger();
+            if (existingRootLogger != null && !existingRootLogger.handlers().isEmpty()) {
+                rootLoggerHandlers.addAll(existingRootLogger.handlers());
+            }
+            rootLoggerHandlers.add(logstashHandler.getKey());
+
             this.logging
                     .customFormatter("logstash", "org.jboss.logmanager.ext", "org.jboss.logmanager.ext.formatters.LogstashFormatter",
                                      this.logstash.formatterProperties())
                     .customHandler(logstashHandler)
-                    .rootLogger(this.logstash.level(), logstashHandler.getKey());
+                    .rootLogger(this.logstash.level(), rootLoggerHandlers.toArray(new String[rootLoggerHandlers.size()]));
```

Comment:
Wondering whether it makes sense for LogStash to have its own level and root logger?

Or instead if we do something like `this.logging.rootLogger().handler(logstashHandler.getKey())`.

Just got me thinking about the relationship between the two logging fractions.